docs/docsite/rst/community/development_process.rst

```diff
@@ -180,31 +180,35 @@ The toplevel keys (bugfixes, major_changes, and so on) are defined in the
   Features that were previously deprecated and are now removed.
 
 **bugfixes**
-  Fixes that resolve issues. If there is a specific issue related to this bugfix, add a link in the changelog entry.
+  Fixes that resolve issues.
 
 **known_issues**
   Known issues that are currently not fixed or will not be fixed.
 
-Most changelog entries will be ``bugfixes`` or ``minor_changes``. When writing a changelog entry that pertains to a particular module, start the entry with ``- [module name] -`` and include a link to the related issue if one exists.
+Each changelog entry must contain a link to its issue between parentheses at the end.
+If there is no corresponding issue, the entry must contain a link to the PR itself.
+
+Most changelog entries will be ``bugfixes`` or ``minor_changes``. When writing a changelog entry that pertains to a particular module, start the entry with ``- [module name] -`` and the following sentence with a lowercase letter.
 
 Here are some examples:
 
 .. code-block:: yaml
 
   bugfixes:
-    - win_updates - fixed issue where running win_updates on async fails without any error
+    - win_updates - fixed issue where running win_updates on async fails
+      without any error (https://github.com/ansible/ansible/issues/29442).
 
 .. code-block:: yaml
 
   minor_changes:
-    - lineinfile - add warning when using an empty regexp (https://github.com/ansible/ansible/issues/29443)
+    - lineinfile - add warning when using an empty regexp (https://github.com/ansible/ansible/issues/29443).
 
 .. code-block:: yaml
 
   bugfixes:
     - copy module - The copy module was attempting to change the mode of files for
-   remote_src=True even if mode was not set as a parameter.  This failed on
-   filesystems which do not have permission bits.
+      remote_src=True even if mode was not set as a parameter.  This failed on
+      filesystems which do not have permission bits (https://github.com/ansible/ansible/issues/29444).
 
 You can find more example changelog fragments in the `changelog directory <https://github.com/ansible/ansible/tree/stable-2.6/changelogs/fragments>`_ for the 2.6 release. You can also find documentation of the format, including hints on embedding rst in the yaml, in the `reno documentation <https://docs.openstack.org/reno/latest/user/usage.html#editing-a-release-note>`_.
```

Comment:
I guess this needs updating as well. 2.6 is pretty old now (though maybe it has better examples), and the link to the reno docs is only partially helpful since a *lot* of it it mentions does not apply (the section names for example).

Reply:
good point!

Reply:
Done. and i've removed the irrelevant link to the reno

Reply:
thanks!